### ISSUE TITLE
[3.0.207] Remix OS TV mode

### ISSUE THREAD

**Issue by lodgnewt** [JideTechnology/Remix-OS-For-PC#28]
Please add a Remix OS TV mode to the system. Thanks

**Comment by jide-bill** [comment]
TV mode is more friendly for remote control. do you have a remote control for PC product? 

Please help us understand more how you would like to use it.

**Comment by lodgnewt** [comment]
Yes i have a remote control rii mini. I use remix on my htpc connect to tv panel for iptv, youtube and other video services apps.

**Comment by ChetanRajpal0204** [comment]
yup really want tv mode in remix os for pc as I have htpc and bluetooth gaming controller, and remote control. would be lot easier if there would be tv mode.

**Comment by Forsskieken** [comment]
I to definately want to choose between launchers....
Let people decide for themselfs if they want stock or leanback (Android TV) launcher.
maybe a toggle switch in settings?

**Comment by Kthulu120** [comment]
Yea TV mode for PC would be pretty glorious

**Comment by luigira** [comment]
TV MODE!!!!! ... Please
